### ISSUE TITLE
Consider extracting all the drivers/libs into standalone repository

### ISSUE THREAD

**Issue by Tvangeste** [CrazyCoder/coolreader-kindle-qt#4]
It would be much easier to reuse them if all the Kindle-specific drivers and libraries were in their own standalone repository rather then inside coolreader's one.

The following are good candidates for extraction:
- Device library
- KindleFiveWay
- KindleKeyboard
- KindleTS
- QKindleFb

**Comment by h1uke** [comment]
Hello Tvangeste,

Monday, December 3, 2012, 7:54:23 AM, you wrote:

T> It would be much easier to reuse them if all the
T> Kindle-specific drivers and libraries were in their own standalone
T> repository rather then inside coolreader's one.

... provided that nobody will be cooking incompatible versions of
mentioned plugins. We were unable to achieve even that so far.

**Comment by CrazyCoder** [comment]
KindleFiveWay is obsolete, KindleKeyboard replaces it. I'll consider making it a separate repository, but it's not a high priority task.

**Comment by CrazyCoder** [comment]
Agreed with @h1uke, libraries change too often right now and there could be incompatibility. It's recommended that every application uses its own copy of these plug-ins/libraries.
